### PR TITLE
Add sections to the consumer concurrency configuration section

### PR DESCRIPTION
**Pull request by thomasgauvin**

### CODE REVIEW

File: content/queues/configuration/consumer-concurrency.md [hunk 1]
```diff
@@ -48,7 +48,14 @@ Cloudflare recommends leaving the maximum concurrency unset, which will allow yo
 
 If you have a workflow that is limited by an upstream API and/or system, you may prefer for your backlog to grow, trading off increased overall latency in order to avoid overwhelming an upstream system.
 
-Concurrency settings can be configured in each projects' `wrangler.toml` file and/or the Cloudflare dashboard. To set concurrency settings in the Cloudflare dashboard:
+You can configure the concurrency of your consumer Worker 2 ways:
```
Comment (kodster28) [inline content/queues/configuration/consumer-concurrency.md]:
```suggestion
To configure the concurrency of your consumer Worker, choose whether to:
```

**Review by kodster28 — APPROVED** [review]

File: content/queues/configuration/consumer-concurrency.md [hunk 2]
```diff
@@ -48,7 +48,14 @@ Cloudflare recommends leaving the maximum concurrency unset, which will allow yo
...
+
+* Set concurrency settings in the Cloudflare dashboard
+* Set concurrency settings via `wrangler.toml`
+
+### Set concurrency settings in the Cloudflare dashboard:
```
Comment (elithrar) [inline content/queues/configuration/consumer-concurrency.md]:
```suggestion
### Dashboard
```

File: content/queues/configuration/consumer-concurrency.md (hunk 1 above)
Comment (elithrar) [inline content/queues/configuration/consumer-concurrency.md]:
```suggestion
You can configure the concurrency of your consumer Worker in two ways:
```

File: content/queues/configuration/consumer-concurrency.md [hunk 3]
```diff
@@ -48,7 +48,14 @@ Cloudflare recommends leaving the maximum concurrency unset, which will allow yo
 
 If you have a workflow that is limited by an upstream API and/or system, you may prefer for your backlog to grow, trading off increased overall latency in order to avoid overwhelming an upstream system.
 
-Concurrency settings can be configured in each projects' `wrangler.toml` file and/or the Cloudflare dashboard. To set concurrency settings in the Cloudflare dashboard:
+You can configure the concurrency of your consumer Worker 2 ways:
+
+* Set concurrency settings in the Cloudflare dashboard
```
Comment (elithrar) [inline content/queues/configuration/consumer-concurrency.md]:
```suggestion
1. Set concurrency settings in the Cloudflare dashboard
```

If you say a number, use a numbered list.

File: content/queues/configuration/consumer-concurrency.md [hunk 4]
```diff
@@ -48,7 +48,14 @@ Cloudflare recommends leaving the maximum concurrency unset, which will allow yo
...
+* Set concurrency settings via `wrangler.toml`
```
Comment (elithrar) [inline content/queues/configuration/consumer-concurrency.md]:
```suggestion
2. Set concurrency settings via `wrangler.toml`
```

File: content/queues/configuration/consumer-concurrency.md (hunk 2 above)
Comment (thomasgauvin) [reply content/queues/configuration/consumer-concurrency.md]: 1. For the sake of consistency, if we change this title to Dashboard, we may want to consider changing the sister title `### Set concurrency settings via `wrangler.toml`` to ``wrangler.toml``. 
2. I believe that the full title makes it easier to grasp and identify what section to read when looking at the docs (see attached pic)

image

Comment (elithrar) [reply content/queues/configuration/consumer-concurrency.md]: That's fine

File: content/queues/configuration/consumer-concurrency.md [hunk 5]
```diff
@@ -48,7 +48,14 @@ Cloudflare recommends leaving the maximum concurrency unset, which will allow yo
 
 If you have a workflow that is limited by an upstream API and/or system, you may prefer for your backlog to grow, trading off increased overall latency in order to avoid overwhelming an upstream system.
 
-Concurrency settings can be configured in each projects' `wrangler.toml` file and/or the Cloudflare dashboard. To set concurrency settings in the Cloudflare dashboard:
+You can configure the concurrency of your consumer Worker in two ways:
+
+1. Set concurrency settings in the Cloudflare dashboard
+2. Set concurrency settings via `wrangler.toml`
+
+### Set concurrency settings in the Cloudflare dashboard:
```
Comment (thomasgauvin) [inline content/queues/configuration/consumer-concurrency.md]:
```suggestion
### Set concurrency settings in the Cloudflare dashboard
```

File: content/queues/configuration/consumer-concurrency.md (hunk 1 above)
Comment (thomasgauvin) [reply content/queues/configuration/consumer-concurrency.md]: Outdated since fixed as per Matt's comments

**Comment by thomasgauvin** [comment]
@elithrar @kodster28 I've adjusted as per your comments.

Kody, you approved but had a comment. I followed Matt's suggestion instead, so please let me know if you're ok with the fix.
Matt, since you own these docs I won't be merging without your 👍, this is a small/low-priority fix and we can leave it until next week anyways.

**Comment by kodster28** [comment]
> @elithrar @kodster28 I've adjusted as per your comments.
> 
> Kody, you approved but had a comment. I followed Matt's suggestion instead, so please let me know if you're ok with the fix. Matt, since you own these docs I won't be merging without your 👍, this is a small/low-priority fix and we can leave it until next week anyways.

Yeah, definitely fine with Matt's version of the fix. We've tried to recruit him to our team before 😛 but haven't had any success yet.

**Comment by elithrar** [comment]
LGTM

**Pull request merged**